packages/framework/src/Model/Customer/User/CustomerUserFacade.php

```diff
@@ -454,4 +455,14 @@ public function getAll(): array
     {
         return $this->customerUserRepository->getAll();
     }
+
+    /**
+     * @param string $uuid
+     * @param \DateTimeInterface $issuedAt
+     * @return bool
+     */
+    public function isIssuedAtStillValid(string $uuid, DateTimeInterface $issuedAt): bool
```

Comment:
I have two "problems" with this:
1) from CustomerUserFacade point of view, it is not clear what `issuedAt` stands for IMHO
2) and this will be a thin ice, it feels like a strictly frontend-api related functionality, should it be in the framework package?

Reply:
maybe it would make sense to rename it to something like `islLastSecurityChangeOlderThan` ... let's discus it :smile:

Reply:
...or we could change it to getter (`getLastSecurityChangeDateTimeByUuid` and add `isIssuedAtStillValid` method to `TokenAutheticator` :thinking:

Reply:
Fair enough, let's discuss it, please... I think it should be in the framework package as the $lastSecurityChange is in the framework also

Reply:
renamed as we discussed - 2f1226ed9f